src/java/org/apache/cassandra/index/sai/StorageAttachedIndexBuilder.java

```diff
@@ -99,7 +99,8 @@ public class StorageAttachedIndexBuilder extends SecondaryIndexBuilder
     @Override
     public void build()
     {
-        logger.debug(logMessage("Starting full index build"));
+        logger.debug(logMessage(String.format("Starting %s %s index build...", isInitialBuild ? "initial" : "non-initial", isFullRebuild ? "full" : "partial")));
```

Comment:
Nit: can we break the long line?
```suggestion
        logger.debug(logMessage(String.format("Starting %s %s index build...",
                                              isInitialBuild ? "initial" : "non-initial",
                                              isFullRebuild ? "full" : "partial")));
```